system/config/templates/features/brokers/brokers.yml

```diff
@@ -27,4 +27,8 @@ registry:
       - key: :allow_edit_broker_email
         is_enabled: false
       - key: :allow_edit_broker_npn
-        is_enabled: false
\ No newline at end of file
+        is_enabled: false
+      - key: :send_broker_hired_event_to_edi
+        is_enabled: <%= ENV['BROKER_HIRED_EVENT_TO_EDI_IS_ENABLED'] || false %>
```

Comment:
do we need to independently turn these on and off? why not just a `BROKER_EDI_EVENT_IS_ENABLED`?